test/call.c

```diff
@@ -412,6 +424,20 @@ static void event_handler(struct ua *ua, enum ua_event ev,
 	case UA_EVENT_CALL_RTCP:
 		++ag->n_rtcp;
 
+		if (f->accept_session_updates &&
+		    call_state(call) == CALL_STATE_ESTABLISHED &&
+		    !call_is_outgoing(call) &&
+		    0 == str_casecmp(prm, "video")&&
+		    sdp_media_ldir(stream_sdpmedia(video_strm(call_video(
+				ua_call(f->a.ua))))) == SDP_SENDRECV &&
+		    sdp_media_rdir(stream_sdpmedia(video_strm(call_video(
+				ua_call(f->a.ua))))) == SDP_SENDRECV &&
+		    sdp_media_ldir(stream_sdpmedia(video_strm(call_video(
+				ua_call(f->b.ua))))) == SDP_SENDRECV &&
+		    sdp_media_rdir(stream_sdpmedia(video_strm(call_video(
+				ua_call(f->b.ua))))) == SDP_SENDRECV) {
```

Comment:
Same here. A state variable.

Reply:
I tried and couldn't get it to work. These events are triggered too often and not when I would expect it.